python/lsst/ts/tunablelaser/hardware.py

```diff
@@ -343,6 +326,27 @@ async def set_output_energy_level(self, value):
         """
         await self.output_energy_level_register.set_register_value(value)
 
+    async def set_propagation_mode(self, value):
+        """Set the propagation mode of the laser.
+
+        value : `str`, {Continuous, Burst, Trigger}
+            The mode to be set
+            Continuous: Continuous pulse
+            Burst: Pulses a set number of times at regular interval
+            Trigger: Trigger a pulse using an external device (Not been used)
```

Comment:
These really should be an enum.

What do you mean by "Not been used"? If we have no trigger input then perhaps say that, e.g. "Not useful for this laser, because we have no trigger source."